hphp/runtime/ext/pdo/ext_pdo.php

```diff
@@ -1,6 +1,15 @@
 <?hh
 // generated by idl-to-hni.php
 
+/**
+ * Objects implementing PDOEntity can handle value by column name and meta
+ */
+interface PDOEntity {
```

Comment:
I think if we stash this in the HH namespace and update the comment to note that it's experimental I'll be fine with adding it. I do have two requests though, (1) add the link to the PHP bug in a comment explaining that this implementation is subject to whatever the PHP community ultimately does, and (2) send an email to the PHP internals mailing list linking to this pull request and explaining your change (just to see if anyone over there wants to pick it up).